### PR TITLE
Random resize and crop

### PR DESCRIPTION
**Pull request by lyuwenyu**
- RandomResize add arg `mode`
  - `long` by default means long side align with maximum
  - `short` means short side align with minimum
- RandomResizeCrop
  - resize as randomresize
  - crop using [min_crop_size, max_crop_size]

### CODE REVIEW

File: ppdet/data/transform/operators.py [hunk 1]
```diff
@@ -1482,6 +1497,155 @@ def apply(self, sample, context=None):
...
+            cover_all_box=cover_all_box,
+            thresholds=thresholds,
+            allow_no_crop=allow_no_crop,
+            is_mask_crop=is_mask_crop)
+
+        self.resizes = resizes
+        self.cropsizes = cropsizes
+        self.prob = prob
+        self.mode = mode
+        self.intetp = interp
```
Comment (ghostxsl) [inline ppdet/data/transform/operators.py]: 应该是self.interp = interp

Comment (lyuwenyu) [reply ppdet/data/transform/operators.py]: done

File: ppdet/data/transform/operators.py [hunk 2]
```diff
@@ -1482,6 +1497,155 @@ def apply(self, sample, context=None):
...
+            is_mask_crop=is_mask_crop)
+
+        self.resizes = resizes
+        self.cropsizes = cropsizes
+        self.prob = prob
+        self.mode = mode
+        self.interp = interp
+        self.keep_ratio = keep_ratio
+
+    def applay(self, sample, context=None):
```
Comment (ghostxsl) [inline ppdet/data/transform/operators.py]: 应该是 apply

Comment (lyuwenyu) [reply ppdet/data/transform/operators.py]: done

**Pull request closed without merging**